docs/backend-api/websocket/subscription.md

```diff
@@ -28,29 +33,30 @@ Request sample:
 ```
 
 ### Response
+
 Response parameters:
 
-* __type__ (required, text: _"response"_).
-* __action__ (required, text: _"subscription/subscribe"_).
-* __events__ (required, enum[], without nulls) - list of the subscribed events. Event can be one of: `state`.
-* __data__ (required, map<string, object>, without nulls) - map with the events subscription result. One key on each subscribed event.
-  * __state__ (presents if the "state" subscription requested, map<string, enum>) - the current status of requested trackers.
+* `type` - required, text: _"response"_.
+* `action` - required, text: _"subscription/subscribe"_.
+* `events` - required, array of string enum, can't be null. List of the subscribed events. Event can be `state`.

```

Comment:
`enum` also described in [data types](https://developers.navixy.com/backend-api/getting-started/#data-types). For json it is `string`, but for our API it is `enum`.
"can't be null" !="without nulls", but "can't be null" == "required" (this also present).